### DISCUSSION TITLE
Mirth 1.8 & windows 2019

### DISCUSSION THREAD

**Discussion by patrizio76** [nextgenhealthcare/connect#5407] [Q&A] [answered]
I can't find documentation, mirth1.8 is compatible with windows 2019?

**Comment by jonbartels** [comment]
You need to post:
* the EXACT version of Mirth you're using
* the EXACT version of Java you're using

Mirth 1.8 was released in approximately 2009. https://forums.mirthproject.io/forum/mirth-connect/general-discussion/2218-mirth-1-8-release-date . This means it would have been targeting Java 6. You're facing a significant risk because Mirth 1.8.x might not run on Java 8 and older versions of Java would not be well supported on a modern OS. 

Mirth has always been built with Install4J. The Service executable for Mirth 1.8 would have targeted Windows versions released around 2009. The service executables might not be compatible.

If you have tried any experiments you should post any errors you get.

**Comment by patrizio76** [comment]
I still don't have anything installed, a customer asks me if it is
possible to install mirth 1.8 on windows server 2019.




Il giorno mar 6 set 2022 alle ore 15:33 Jon Bartels
***@***.***> ha scritto:

**Reply by jonbartels** [reply]
Installing Mirth 1.8 to Windows 2019 might be possible but it is a horrible idea. Mirth 1.8 is 13 years old and may not run on a modern Java release. 

A better solution would be:
* Have an export of the entire Mirth 1.8 database
* Install a modern Java version to Windows (Java 11 or Java 17)
* Install a modern Mirth Connect version (4.1.0)
* Using the Mirth 1.8 database export, start extracting and converting channels to Mirth 4.1.0

This project is possible but you either have to get 13 year old software running on a modern OS *or* convert an old Mirth installation to a new Mirth installation.

**Comment by tonygermano** [comment, marked as answer]
I would say the simple answer is that mirth 1.8 is not supported, so getting it running would be best effort.
